src/modules/dashboard/components/ExpenditurePage/TitleDescriptionSection/TitleDescriptionSection.css

```diff
@@ -29,9 +29,15 @@
 }
 
 .titleContainer {
+  display: flex;
+  align-items: center;
   margin-bottom: 21px;
 }
 
+.titleContainer div[class*="Input_container"] {
```

Comment:
Could you please explain which element you're trying to target here?

Reply:
i wanted to get to the child component node in this place

![Screenshot 2022-10-07 at 11 01 31](https://user-images.githubusercontent.com/93312086/194526486-79f33736-5457-4dbc-bd35-07019ad613cb.png)

Reply:
Makes sense, thanks. You might want to consider adding a comment here